test/test_versions.py

```diff
@@ -28,13 +28,11 @@
 
 from picard import (
     api_versions,
-    api_versions_tuple,
-    version_from_string,
-    version_to_string,
+    api_versions_tuple
```

Comment:
Just a minor thing, but please keep the comma at the end of this line.
It prevents a change if we had or remove a line in this import.

Reply:
Fair point, I'll keep that in mind.